guides/doc-Deploying_Project_on_AWS/master.adoc

```diff
@@ -9,17 +9,19 @@ ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
-include::common/modules/con_scenarios-for-foreman-on-aws.adoc[leveloffset=+1]
+#Commenting to avoid build issues. This guide will be deleted soon.
+#include::common/modules/con_scenarios-for-foreman-on-aws.adoc[leveloffset=+1]
```

Comment:
AsciiDoc syntax for comments is `// comment`. [Reference](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#comments).

But if this guide will be removed, I think that you can just delete the rename includes.